src/SELF_DGModel1D_t.f90

```diff
@@ -338,9 +335,9 @@ subroutine Write_DGModel1D_t(this,fileName)
 
     INFO("Writing pickup file : "//trim(pickupFile))
     call this%solution%UpdateHost()
-    if(this%decomp%mpiEnabled) then
+    if(this%mesh%decomp%mpiEnabled) then
 
-      call Open_HDF5(pickupFile,H5F_ACC_TRUNC_F,fileId,this%decomp%mpiComm)
+      call Open_HDF5(pickupFile,H5F_ACC_TRUNC_F,fileId,this%mesh%decomp%mpiComm)
```

Comment:
This line, and many below, are parallel IO. Since I don't plan to support domain decomposition for 1-D models (it's a bit overkill), I'll go through and trim out all mpi related business for 1-D components.

Reply:
Resolved with [4f7a7a0](https://github.com/FluidNumerics/SELF/pull/53/commits/4f7a7a0e157b85dc677092583d328816442ade81)